src/scheduler/server_info.cpp

```diff
@@ -750,21 +751,10 @@ query_server_dyn_res(server_info *sinfo)
 					log_eventf(PBSEVENT_DEBUG, PBS_EVENTCLASS_SERVER, LOG_DEBUG, "server_dyn_res",
 					"Program %s timed out", conf.dynamic_res[i].command_line);
 				}
-				if (ret == -1 || ret == 0) {
-					/* we know pid is initialized here, otherwise pipe_err would have been set */
-					kill(-pid, SIGTERM);
-					if (waitpid(pid, NULL, WNOHANG) == 0) {
-						usleep(250000);
-						if (waitpid(pid, NULL, WNOHANG) == 0) {
-							kill(-pid, SIGKILL);
-							waitpid(pid, NULL, 0);
-						}
-					}
-					pid = 0; /* don't kill it again later */
-				}
-				/* Parent; assume fdopen can't fail, but only open if child was not killed. */
-				k = 0;
-				if (pid > 0) {
+				if ( pid > 0 && ret > 0 ) {
```

Comment:
Remove the space after ( and before )

Reply:
Done (although personally I find that convention jarring when you have operators that _are_ separated from their arguments with spaces, which gives a counterintuitive visual grouping, it is what it is).

Reply:
In another life, someone insisted on the if( blah >= 0 ) style...so I'm not religious about these things ;-).